### ISSUE TITLE
Delete Thumbnail via Delete Key

### ISSUE THREAD

**Issue by jauyong** [GoogleForCreators/web-stories-wp#1919]
## Feature Description

I should be able to select a thumb and hit delete

UX needed to indicate selected thumbnail state. Currently there is only a blue bar above the thumbnail to indicate which page is displayed on the canvas

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance Criteria

1. After selecting a thumbnail, I should be able to delete the page via the delete key

**Comment by samitron7** [comment]
@jauyong Can you clarify if this is focus state for kb navigation? If not, the blue bar is sufficient 
enough to indicate that's the thumbnail you want to delete and we can close this ticket.

**Comment by o-fernandez** [comment]
Unclear that we need this, I will close for now and we can reopen if truly needed.
